.github/workflows/test-packaging-python.yml

```diff
@@ -0,0 +1,60 @@
+#
+# DO NOT EDIT THIS FILE: it is generated from test-transform.template,  Edit there and run make to change these files
```

Comment:
Probably should remove these comments since these are not auto generated.

Reply:
Done! Thanks